ctapipe/io/hdf5eventsource.py

```diff
@@ -351,18 +349,6 @@ def _parse_sb_and_ob_configs(self):
 
         return scheduling_blocks, observation_blocks
 
-    def _is_hillas_in_camera_frame(self):
```

Comment:
Just to be sure:
We want to completely drop reading files created in the CameraFrame?

Reply:
I think so. Like you said, it will happen at some point. In this case it is just the result of getting rid of `CameraHillasParametersContainer`